### ISSUE TITLE
Unable to save study on Ubuntu

### ISSUE THREAD

**Issue by hugo-antoine-rtei** [AntaresSimulatorTeam/Antares_Simulator#479]
**Description**
When trying to save a study (new or loaded), I get a window saying:

> **Impossible to save the study at this location**
> Please choose another folder to save the study.

I get this message regardless of the folder I put it in (at /home/user it doesn't work either). I also tried using sudo before launching antares.

**How to Reproduce**

- Download the .deb of the 8.1 https://github.com/AntaresSimulatorTeam/Antares_Simulator/releases/download/v8.1.0/antares-8.1.0-Ubuntu-20.04.deb
- Install it
- Launch it using the command `antares-8.1-ui-simulator`
- Start an empty study and try to save

**Expected behavior**
A saved study

**Screenshots and additional files**
Screenshot of the window I get:
![Capture d’écran du 2021-12-08 18-57-40](https://user-images.githubusercontent.com/73881097/145259376-c67f6804-8951-43ad-bae6-37aa692352c7.png)

**OS and version**
 - OS: Ubuntu 20.04.3
 - Antares Simulator version: 8.1

**Additional context**
When building antares from scratch, saving a study works fine.

**Comment by flomnes** [comment]
See **src/ui/simulator/windows/saveas.cpp**
```cpp
        String pathCopy = path;
        pathCopy.toLower();

        if (pathCopy.startsWith(rootFolder))
        {
            Window::Message message(&mainFrm,
                                    wxT("Save As"),
                                    wxT("Impossible to save the study at this location"),
                                    wxString()
                                      << wxT("Please choose another folder to save the study."));
            message.add(Window::Message::btnCancel);
            message.showModal();
            return;
        }
```
The problem seems to be that `pathCopy.startsWith(rootFolder)` is evaluated to `true`.

**Comment by flomnes** [comment]
Duplicate of #598

Closed by #600
